protocols/relay/src/v2/client/transport.rs

```diff
@@ -320,12 +320,21 @@ fn parse_relayed_multiaddr(
 
 pub struct RelayListener {
     listener_id: ListenerId,
+    /// Queue of events to report when polled.
     queued_events: VecDeque<<Self as Stream>::Item>,
+    /// Channel for messages from the behaviour [`Handler`][super::handler::Handler].
```

Comment:
```suggestion
    /// Channel for messages from the behaviour [`Client`][super::Client].
```

Why reference the handler here? Wouldn't the `NetworkBehaviour` implementation be more appropriate?

Reply:
Thanks for elaborating! The above makes sense. No opinion on what is the simpler approach. Fine leaving it as is.